docs/product-updates/2023-08-01-custom-code-samples.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Custom code samples
+tags: [Improvement]
+image: /files/changelog/code-samples-support-openapi.png
+---
+
+![bump-sh-api-branch-management.png](/files/changelog/code-samples-support-openapi.png)
+
+We now support the `x-codeSamples` extension that let you set language-specific code samples you expect your users to work with.
+
+Examples are often the first stop for developers in any API documentation. This extension allows you to set along a programming language, its custom code samples, for a more seamless developer experience when it comes to testing an API.
```

Comment:
@antlio shall we add a link to https://docs.bump.sh/help/doc-code-samples/ somewhere in the text?

Reply:
I did on the last sentence 👍